task6.py

```diff
@@ -0,0 +1,10 @@
+'''
+Find how many number of times the substring occurs in the given string.
+ Input:
+string = “PQRQRQRQ” substring = “QRQ”
+ Output: 3
+
+'''
+import re
+string = "PQRQRQRQ"
+print(len(re.findall('(?=QRQ)', string)))
```

Comment:
Good! Please explain me how this works.

Reply:
findall method find all which expression present in string ?= match all QRQ string and find length